manifests/init.pp

```diff
@@ -40,7 +40,6 @@
   $conn_keep_alive          = $puppetdb::params::conn_keep_alive,
   $conn_lifetime            = $puppetdb::params::conn_lifetime,
   $puppetdb_package         = $puppetdb::params::puppetdb_package,
-  $puppetdb_version         = $puppetdb::params::puppetdb_version,
```

Comment:
So you've removed the ability to tweak this on a per class basis from puppetdb & puppetdb::server, but not from puppetdb::master::config, what direction were you thinking globally?